data/battle_4.s

```diff
@@ -7,44 +7,134 @@
 
 
 gMoveEffectBS_Ptrs:: @ 831C224
-	.incbin "baserom.gba", 0x31c224, 0xa4
-
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB386
+	.4byte gUnknown_082DB395
+	.4byte gUnknown_082DB3A4
+	.4byte gUnknown_082DB3B3
+	.4byte gUnknown_082DB3C9
+	.4byte gUnknown_082DB3E6
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB3C2
+	.4byte gUnknown_082DB3D6
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB3DD
+	.4byte gUnknown_082DB3F4
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB3F4
+
+	.align 2
+	window_template 0x00, 0x01, 0x03, 0x07, 0x0f, 0x1f, 0x003f
```

Comment:
is this really a window template?

Reply:
Not 100% sure but it fits perfectly with the pattern I've consistently seen for all window templates.